julia/test/MyGame/Example/Ability.jl

```diff
@@ -0,0 +1,14 @@
+# automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Please move these files to be in `test/` to be analogous to all other languages.

Reply:
ahh, looks like they're simply duplicates, so remove them here.

Reply:
Done!